remix-tests/src/compiler.ts

```diff
@@ -21,6 +21,32 @@ function writeTestAccountsContract (accounts: string[]) {
     return testAccountContract.replace('>accounts<', body)
 }
 
+function processFile(filePath: string, sources: any, isRoot: boolean = false) {
+    try{
+        const importRegEx = /import ['"](.+?)['"];/g;
+        let group: any ='';
+        if(filePath.includes('tests.sol') || filePath.includes('remix_tests.sol') || filePath.includes('remix_accounts.sol') || Object.keys(sources).includes(filePath))
+            return
+
+        let content = fs.readFileSync(filePath, { encoding: 'utf-8' });
+        const s = /^(import)\s['"](remix_tests.sol|tests.sol)['"];/gm
+        if (isRoot && filePath.indexOf('_test.sol') > 0 && regexIndexOf(content, s) < 0) {
```

Comment:
Prefer `filePath.includes('_test.sol')` to `indexOf` as the semantic is more understandable for reader.